### PR TITLE
Better error msg for validity check of geography

### PR DESCRIPTION
**Pull request by jievince**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [ ] enhancement

#### Description:
When geography dataum is found to be illegal, print the specific cause of the error to facilitate troubleshooting.
eg.
```
E20220902 03:24:24.550307 350836 FunctionManager.cpp:2440] ST_GeogFromText error: The geography LINESTRING(0 1, 2 3) built from wkt LINESTRING(0 1, 2 3) is invalid, error: FindValidationError, S2 Errorcode: 1, message: Vertex 0 is not unit length
```

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Review by Aiee — DISMISSED** [review]

File: src/common/function/FunctionManager.cpp [hunk 1]
```diff
@@ -2511,7 +2511,8 @@ FunctionManager::FunctionManager() {
       if (!args[0].get().isGeography()) {
         return Value::kNullBadType;
       }
-      return args[0].get().getGeography().isValid();
+      auto status = args[0].get().getGeography().isValid();
+      return status.ok() ? true : false;
```
Comment (Shylock-Hg) [inline src/common/function/FunctionManager.cpp:2515]: It's seems not affect.

**Review by czpmango — APPROVED** [review]
LGTM

File: src/common/function/FunctionManager.cpp (hunk 1 above)
Comment (czpmango) [inline src/common/function/FunctionManager.cpp:2515]: return status.ok()

Comment (jievince) [reply src/common/function/FunctionManager.cpp]: Missed it here, thanks

Comment (jievince) [reply src/common/function/FunctionManager.cpp]: Fixed

**Review by Aiee — APPROVED** [review]

**Review by czpmango — APPROVED** [review]

**Pull request merged**